index.js

```diff
@@ -0,0 +1,4 @@
+document.getElementById("submit").onclick = function (){
```

Comment:
The function is not completed. Kindly completed it else if it is not needed, kindly take it out of the code to make your work clean